src/lua/init.c

```diff
@@ -94,6 +95,14 @@ luaopen_zip(lua_State *L);
 
 #define MAX_MODNAME 64
 
+static bool background = false;
```

Comment:
I think it's better to rename it to `is_background` or something like that since the variable is boolean.